src/shogun/statistical_testing/MMD.cpp

```diff
@@ -136,6 +139,16 @@ index_t CMMD::get_num_null_samples() const
 	return self->num_null_samples;
 }
 
+void CMMD::set_random_seed(int32_t p_seed)
+{
+	self->seed = p_seed;
+}
+
+index_t CMMD::get_random_seed() const
+{
+	return self->seed;
+}
+
 void CMMD::set_statistic_type(EStatisticType stype)
```

Comment:
Hi  @karlnapf and @lambday, my question is like this:

In kernlelselect and other places related to CrossValidation, we need to set a specific random seed to make it can generate same result every time. Usually, before we remove sg_rand, it can been done like[ sg_rand->set_seed(12345)](https://github.com/shogun-toolbox/shogun/pull/3888/files#diff-ade8fd54db951771ca40a368b7e792deL53). Then it will pass to the CrossValidations like here[https://github.com/shogun-toolbox/shogun/pull/3888/files#diff-b4b05fef7370d38882f73b554c6230ebL19]. 

But after we remove the global random, it become a little difficult to make that happened, I think. Because in kernel selection test, we can't directly access its cv object like we do in [here](https://github.com/shogun-toolbox/shogun/pull/3888/files#diff-727ce9dc4098d551004a7bf7d6ce8c43R304) to set its seed. 

So I add one more parameter which is the [**seed**](https://github.com/shogun-toolbox/shogun/pull/3888/files#diff-1aeca6bef489a2963141948c747855f5R54), and make it default return an fixed seed(**12345**) and pass it to  MaxCrossValidation like [this]( https://github.com/shogun-toolbox/shogun/pull/3888/files#diff-e76beae1ecab5719393a08a37a5a5ef8R106). 

But I'm not sure if it's appropriate to return a fixed seed a default when we do kernel select, and I'm kind of doesn't understand why we [choose 0.25](https://github.com/shogun-toolbox/shogun/pull/3888/files#diff-ade8fd54db951771ca40a368b7e792deR290) as kernel width rather than others.